testfiles/1ZoneUncontrolled_Win_ASH55_Thermal_Comfort.idf

```diff
@@ -444,12 +444,12 @@
 
   People,
     ZONE ONE People,         !- Name
-    ZONE ONE,                !- Zone or ZoneList Name
+    ZONE ONE,                !- Zone or ZoneList or Space or SpaceList Name
```

Comment:
This field name change needs to be described in Rules22-1-0-to-22-2-0.md

Reply:
Same for People, Lights, and all *Equipment object. The field names changed before, this just brings the example files up to date, because I ran the entire set through transition so it cleaned up all the field names.

Reply:
OK, I see. So these would show up in a previous change log.

Reply:
> OK, I see. So these would show up in a previous change log.

Correct. https://github.com/NREL/EnergyPlus/blob/develop/src/Transition/InputRulesFiles/Rules9-5-0-to-9-6-0.md